Source/WebCore/contentextensions/ContentExtensionParser.h

```diff
@@ -27,6 +27,7 @@
 
 #if ENABLE(CONTENT_EXTENSIONS)
 
+#include "CSSParserContext.h"
```

Comment:
This is likely not needed.  Can you forward declare CSSParserContext instead?

Reply:
I tried that when working on the patch and ran into a build error.